servers/rendering/renderer_scene_cull.cpp

```diff
@@ -1699,20 +1662,12 @@ uint32_t RendererSceneCull::get_pipeline_compilations(RS::PipelineSource p_sourc
 }
 
 void RendererSceneCull::instance_geometry_get_shader_parameter_list(RID p_instance, List<PropertyInfo> *p_parameters) const {
+	ERR_FAIL_NULL(p_parameters);
 	const Instance *instance = instance_owner.get_or_null(p_instance);
-	ERR_FAIL_NULL(instance);
```

Comment:
I.e. restore this, as it is not guaranteed to be valid

Reply:
Just to clarify: You mean remove it or add it?

Reply:
Put it back the way it was

Reply:
Alright. Thanks for clarifying. Should be done now I think